src/CubicSplines.jl

```diff
@@ -79,4 +79,34 @@ function (A::CubicSpline{<:AbstractVector{<:Number}})(t::Number)
     I + C + D
 end
 
+function cumulative_integral!(out, A::CubicSpline, t1, t2)
```

Comment:
I know this is mostly just a helper function, but could you add a small comment expressing your assumptions about the length of `out`?